crates/starknet-types-core/src/felt/mod.rs

```diff
@@ -1084,6 +1090,22 @@ mod test {
     #[cfg(feature = "serde")]
     use serde_test::{assert_de_tokens, assert_ser_tokens, Configure, Token};
 
+    #[test]
+    fn test_debug_format() {
+        assert_eq!(
+            format!("{:?}", Felt::ONE),
+            "0x0000000000000000000000000000000000000000000000000000000000000001"
```

Comment:
If you use `repeat` it's easy to see the number of leading zeros is as expected
```suggestion
            "0x" + std::iter::repeat("0").take(63).collect::<String>() + "1"
```

Reply:
Done.